app/views/project_mailer/delete_project_completed.text.erb

```diff
@@ -31,3 +31,12 @@ See COPYRIGHT and LICENSE files for more details.
 <%= project_url(@project) %>
 
 <%= t('projects.delete.completed_text', name: @project.name) %>
+
+<% if @project.descendants.any? %>
```

Comment:
I think `@project.descendants` cannot work once the subprojects are deleted. It will return other results including existing projects.